.readthedocs.yaml

```diff
@@ -17,15 +17,18 @@ build:
       # Build package with doc requirements from pyproject.optional-dependencies
       - pip install --verbose --editable .[doc]
       # remove API DLL define in header
-      - find power_grid_model_c/power_grid_model_c/include  -name *.h -exec sed -i -r "s/#define PGM.*//g" {} \;
-      - find power_grid_model_c/power_grid_model_c/include  -name *.h -exec sed -i -r "s/PGM_API //g" {} \;
+      - find power_grid_model_c/power_grid_model_c/include -name *.h -exec sed -i -r "s/#define PGM.*//g" {} \;
+      - find power_grid_model_c/power_grid_model_c/include -name *.h -exec sed -i -r "s/PGM_API //g" {} \;
       # build doxygen for C header
       - cd docs/doxygen && doxygen && cd ../..
       # download support
       - wget -P docs/release_and_support https://github.com/PowerGridModel/.github/raw/main/RELEASE.md
       - wget -P docs/release_and_support https://github.com/PowerGridModel/.github/raw/main/SUPPORT.md
+      - wget -P docs/release_and_support https://github.com/PowerGridModel/.github/raw/main/SECURITY.md
```

Comment:
alternatively, we could also add a copy of the security policy to this repository itself. then the security policy button in the right sidebar of the GitHub GUI will also work correctly